applications/vessel-history/src/App.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment, useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import GFWAPI from '@globalfishingwatch/api-client'
+import useGFWLogin from '@globalfishingwatch/react-hooks/dist/use-login'
+import Home from 'features/home/Home'
+import { getLocationType } from 'routes/routes.selectors'
+import { PROFILE } from 'routes/routes'
+import { BASE_URL, SPLASH_TIMEOUT } from 'data/constants'
+import Profile from 'features/profile/Profile'
+import Splash from 'components/Splash'
+import './App.css'
+
+GFWAPI.setConfig({ dataset: process.env.REACT_APP_DATASET || 'carriers:v20200820' })
```

Comment:
Legacy GFWAPI stuff... we don't use a fixed version of the dataset anymore so we could drop it off

Reply:
This is removed in #297